app/components/publish-actions.tsx

```diff
@@ -20,15 +12,14 @@ import {
 } from "@mui/material";
 import { ChangeEvent, ReactNode, RefObject, useEffect, useState } from "react";
 
-
 import { CHART_RESIZE_EVENT_TYPE } from "@/charts/shared/use-size";
 import { CopyToClipboardTextInput } from "@/components/copy-to-clipboard-text-input";
-import Flex from "@/components/flex";
+import { default as Flex } from "@/components/flex";
```

Comment:
Extra-nit: I think it's a bit easier to not do `{ default as Flex }`, but simply `Flex` 😅 But might be a personal preference :)

Reply:
Auto formatting added this idk why.